Code Examples/Info Tab Example.nlogo

```diff
@@ -269,32 +303,52 @@ To include a short piece of code in a sentence, surround it with backticks (`).
 
 #### Example
 
-    You can create a single turtle with the `crt 1` command.
+```text
+You can create a single turtle with the `crt 1` command.
+```
 
 #### Formatted
 
 You can create a single turtle with the `crt 1` command.
 
 ## Code blocks
 
-It is also possible to have blocks of code. To create a code block, indent every line of the block by 4 spaces.
+It is also possible to have blocks of code. To create a code block, indent every line of the block by 4 spaces. Another way is to sorround it with a three back-ticks before and after the block. (If you don't want your code to be colored as NetLogo code, add `text` after the first three back-ticks.)
```

Comment:
- [ ] Typo: "sorround".
- [ ] "back-tick" should be one word: "backtick"
- [ ] "a three back-ticks": remove the "a"